cli/src/semgrep/meta.py

```diff
@@ -545,7 +570,10 @@ def branch(self) -> Optional[str]:
         """
         if self.event_name == "pull_request_target":
             return os.getenv("GITHUB_HEAD_REF")
-        return os.getenv("GITHUB_REF")
+        github_ref = os.getenv("GITHUB_REF")
+        if github_ref:
+            return github_ref
+        return super().branch
```

Comment:
if we want to keep behavior consistent shouldn't we return SEMGREP_BRANCH before GITHUB_REF?

Reply:
yep, that's fair, fixed